functorch/_src/eager_transforms.py

```diff
@@ -41,7 +41,7 @@ def unwrap_tensors(x):
         if isinstance(x, tuple):
             return tree_map(unwrap_tensors, tuple(x))
 
-        raise RuntimeError(f"Expected tensors, got unsupported type {type(x)}")
+        return x
```

Comment:
This might change how grad/vjp handle outputs as well. I'm not sure we actually have tests for this

Reply:
IMO, yes, I think as well that there is no test for this line